### ISSUE TITLE
multi-touch support

### ISSUE THREAD

**Issue by gase12** [visjs/vis-network#46]
hi feature request: 
would be great to be able to add multitouch support on network visualizations. Dragging multiple nodes would give a nice ux boost. 
if this can be done in current versions, please ignore, i haven't found it.
best. g

**Comment by mojoaxel** [comment]
We already habe pinch-zoom with two fingers. I'm not sure how we should distinguish between pinch-zoom and multi-drag!?

**Comment by gase12** [comment]
Hi. 
I am thinking of 2 options: (only on nodes) A: either pich zoom or multidrag supported. Not both at the same time. Option on init. 

Or B: -better- if drag event fires on either touch input the next one would not trigger pinch. (If a Node touched drag will occur)

**Comment by Thomaash** [comment]
I don't know. If I pinch I want to zoom no matter where I pinch, not carefully look and pick a good location where I can pinch to zoom without moving something by accident.

**Comment by gase12** [comment]
yes, thats breaking zoom-pinch in that case a bit.  - at least if the screen is full of nodes
(maybe could be linked to zoomlevel, but that is quite bad ux :) )

in my usecase i dont need zoom and looking for a solution for a touch based kiosk app. thats why i asked around.. 
so any lead would be welcome.. hackish solutions either... 
thanks.

**Comment by Thomaash** [comment]
Well, my comment doesn't apply if zoom is disabled, that's okay with me.

**Comment by gase12** [comment]
ok. so i poke around in InteractionHandler.js to see if it is doable.. i guess just ignoring the pinch event first could be a start...
